src-tauri/src/lib.rs

```diff
@@ -114,16 +133,6 @@ async fn create_calendar(
     let calendar_id = hash.into();
     let topic = NetworkTopic::Calendar { calendar_id };
 
-    // @TODO: It would be clearer if this "selecting" of the calendar was handled by calling the
```

Comment:
If we remove this section of code from the backend (which we want to do now that replay is possible) we should add a call to `select_calendar` in the frontend so that the newly created calendar is selected.

Reply:
Maybe either already add that in this PR or open an issue.

Reply:
Okay, I've added that now in the frontend! Doesn't feel good where it is yet, but that's fine!